scripts/check_imports.swift

```diff
@@ -34,6 +34,7 @@ let skipDirPatterns = ["/Sample/", "/Pods/",
   [
     "CoreOnly/Sources", // Skip Firebase.h.
     "SwiftPMTests", // The SwiftPM imports test module imports.
+    "FirebaseSessions/Protogen/", // Generated nanopb code with imports
```

Comment:
@paulb777, is there a way to do this without excluding from the `check_import` check?